modules/meeting/app/views/meetings/_form.html.erb

```diff
@@ -191,4 +191,16 @@ See COPYRIGHT and LICENSE files for more details.
   </div>
 
   <%= render partial: 'meetings/participants_section' %>
+
+  <div class="form--field">
+    <%= styled_label_tag 'send_notfications', t(:"meeting.email.send_emails") %>
+    <div class="form--field-container">
+      <%= styled_check_box_tag 'send_notifications',
+                               1,
+                               true %>
+    </div>
+    <div class="form--field-instructions">
+      <%= t(:"meeting.email.send_invitation_emails") %>
+    </div>
+  </div>
```

Comment:
For a better user experience, I would hide this field from the edit form. Show it only on the create and copy form.